Altalite.py

```diff
@@ -7,35 +7,43 @@
 
 collections.Callable = collections.abc.Callable
 
-print("---------------------------------------------------------------------------------")
-print("Welcome in app Altalite!\n"
-      "Enter below the ticker of the company you are interested in and I will analyze it for you")
-print("---------------------------------------------------------------------------------")
 
-CompanyTicker = input("Enter company ticker:")
+CompanyTicker = input("Enter company ticker you want analyze:")
+
+
+def DisplayTitle():
+    print(Separator())
```

Comment:
Nie musisz wrzucać tego w print, skoro jest on wewnątrz metody ```Separator()``` (to by trzeba wszędzie pozmieniać)

Reply:
Wrzuciłem commita

Reply:
Jeżeli zrobiłeś poprawkę dokładnie jak w sugestii z komentarza, to klikaj po prostu `Resolve conversation`